skyportal/handlers/api/comment.py

```diff
@@ -1298,7 +1291,6 @@ def get(self, associated_resource_type, resource_id, comment_id):
                 "Cannot set both download and preview to True. Please set only one to True, or set both to False."
             )
 
-        table, resource_id_col = None, None
```

Comment:
it's always good pratice to initialize variables before a set of if statements. I know that in python there is no concept of scope in a function, but it does help avoid errors if a variable does not created by a set of if/else/elif statements. I'm fine removing it but honestly, code is safer and more readable with variables clearly defined and not just conditionally.